src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -617,14 +617,17 @@ int wm_vuldet_compare_pkg(cve_vuln_pkg *Pkg1, cve_vuln_pkg *Pkg2) {
         return -1;
     }
 
-    if (strcmp(Pkg1->version, Pkg2->version)) { // Version
+    if (!(!Pkg1->version && !Pkg2->version) &&
```

Comment:
@0xN3utr0n could you confirm that the addition of this conditions again is correct? The removal of those conditions was causing some failures in the unit tests and also could result in some crashes.